src/studio/src/designer/backend/Services/Implementation/TextsService.cs

```diff
@@ -49,5 +62,46 @@ public void DeleteTexts(string org, string repo, string developer, string langua
 
             altinnAppGitRepository.DeleteTexts(languageCode);
         }
+
+        /// <summary>
+        /// Helper method for extracting the markdown filenames from values in a texts objects.
+        /// </summary>
+        /// <param name="texts">Json object consisting of texts with key:value pairs id:text</param>
+        /// <returns>List of markdown filenames</returns>
+        private static List<string> ExtractMarkdownFileNames(Dictionary<string, string> texts)
+        {
+            List<string> markdownFileNames = new List<string>();
+            foreach (KeyValuePair<string, string> text in texts.Where(text => text.Value.StartsWith("${{") && text.Value.EndsWith(".md}}")))
```

Comment:
This is nit picking level, but when you have expressions like `text.Value.StartsWith("${{") && text.Value.EndsWith(".md}}`consider creating a small method stating the intention of what you actually are stating. This for example could be `texts.where(text => IsFileReference(text.Value))` (might be wrong syntax) but this you can just read without thinking. But like I said, nit picking level and not the most complex statement:)